.travis.yml

```diff
@@ -49,7 +49,8 @@ matrix:
                SPLIT=0
         - os: linux
           env: PYTHON_VERSION=2.7
-               CONDA_DEPENDENCIES="numpy=1.8 scipy=0.12 matplotlib=1.3 pandas=0.13 scikit-learn=0.15 nose pytest pytest-cov"
+               CONDA_DEPENDENCIES="numpy=1.8 scipy=0.12 matplotlib=1.3 pandas=0.13 scikit-learn=0.15 nose pytest pytest-cov numexpr"
+               PIP_DEPENDENCIES="python-picard"
```

Comment:
This should not be necessary. The failure on the Travis run from my commit is indicative that if someone has `sklearn` installed but not `picard` then tests are not properly skipped. It would be better to leave this run alone (so it is in a has-sklearn does-not-have-picard state) and fix that problem.

You can debug locally easily by temporarily removing `picard` from your system.

Reply:
Thanks for the tip! I think it's fixed now :)